requirements.txt

```diff
@@ -368,6 +368,6 @@ zope-interface==5.4.0
     #   persistent
     #   transaction
     #   zodb
-
+pyopenair==1.1.0
```

Comment:
you need to use pip-tools to generate requirements file. see docs/CONTRIBUTING.rst and development section